spec/faked_project/lib/faked_project/some_class.rb

```diff
@@ -11,11 +11,9 @@ def reverse
   end
 
   def compare_with(item)
-    if item == label
-      return true
-    else
-      fail "Item does not match label"
-    end
+    return true if item == label
```

Comment:
That would be the [`Style::GuardClause` cop](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/GuardClause), which says:
```
Use a guard clause instead of wrapping the code inside a conditional expression.
```